joint_state_publisher_gui/joint_state_publisher_gui/joint_state_publisher_gui.py

```diff
@@ -61,6 +61,21 @@
 
 RANGE = 10000
 
+LINE_EDIT_WIDTH = 40
+SLIDER_WIDTH = 200
+
+INIT_NUM_SLIDERS = 7  # Initial number of sliders to show in window
```

Comment:
Are the line separations necessary? Unify under one commentline?

Reply:
done